Added a Horizon Indicator

atlantichalibut opened the pull request:
Created a new GameObject. Created HorizonRenderer script, added it to the new GameObject

It seems like it added a LOT of code to viewer.unity I don't know if this is correct or not

(Let me know if It's broken for some reason then I can try re-doing this. I HOPE this works)

GingerSteve commented on Assets/Scripts/Model/HorizonRenderer.cs:
```diff
@@ -0,0 +1,58 @@
+public class HorizonRenderer : MonoBehaviour {
+
+
+    public float width = 90;
```
Please take a look at [the code style guideline](https://github.com/GingerSteve/Stargazing-App/wiki/Code-Style-Guideline) – public members should use PascalCase.

atlantichalibut commented:
Ok, I've made those changes (apologies for the really bad commit title).

I fixed up the obsolete functions.
Changed the number of vertices to 50 any lower begins to get kinda jank.
Created a new material and added it as a component to the Horizon game object.
Changed variable names.

GingerSteve commented:
@atlantichalibut Looks good, good job!
This is now merged into master.
